.murdock

```diff
@@ -107,6 +107,17 @@ get_supported_kconfig_board_app() {
     return 1
 }
 
+kconfig_module_packages_diff() {
+    local board=$1
+    local appdir=$2
+    make clean --no-print-directory -C ${appdir} BOARD=${board}
+    local make_modules=$(make --no-print-directory info-modules info-packages -C ${appdir} BOARD=${board})
+    make clean --no-print-directory -C ${appdir} BOARD=${board}
+    local kconfig_modules=$(TEST_KCONFIG=1 make --no-print-directory info-modules info-packages -C ${appdir} BOARD=${board})
+
+    diff <(echo "${make_modules}") <(echo "${kconfig_modules}")
```

Comment:
I guess I cannot use this with the .murdock scripts...  I will just write a tmp_file or something and do a diff on that...